openlane_run/action.yml

```diff
@@ -0,0 +1,120 @@
+name: Run design with OpenLane.
+
+description: >
+  The goal of this action is to run a given design through the OpenLane flow
+  using a given tag. Optionally a PR can be created to update a stable tag
+  file stored in the repository.
+
+inputs:
+  ol_tag:
+    description: OpenLane tag. You can set to ol_tag_file to use the value from the input.
+    default: master
+  update_tag:
+    description: If the test is successful update stable tag pointed by the ol_tag_file input.
+    default: false
+  ol_tag_file:
+    description: File where to store the new OpenLane tag
+    default: .github/openlane-stable-tag
+  tools_list:
+    description: List of the tools to update separated by a space. Example "openroad_app magic".
+    default: None
```

Comment:
should the default be an an empty string?

Reply:
Fixed on [0b11b71](https://github.com/The-OpenROAD-Project/actions/pull/11/commits/0b11b7106247798cb27386930a42419f6d632d64)